src/archivematicaCommon/lib/executeOrRunSubProcess.py

```diff
@@ -145,26 +145,25 @@ def createAndRunScript(
         arguments = []
     if env_updates is None:
         env_updates = {}
-    # Output the text to a /tmp/ file
-    scriptPath = "/tmp/" + uuid.uuid4().__str__()
-    FILE = os.open(scriptPath, os.O_WRONLY | os.O_CREAT, 0o770)
-    os.write(FILE, text.encode("utf8"))
-    os.close(FILE)
-    cmd = [scriptPath]
-    cmd.extend(arguments)
-
-    # Run it
-    ret = launchSubProcess(
-        cmd,
-        stdIn="",
-        printing=printing,
-        env_updates=env_updates,
-        capture_output=capture_output,
-    )
-
-    # Remove the temp file
-    os.remove(scriptPath)
-
+    # Output the text to a temporary file
```

Comment:
Could we move this comment below? Maybe above the `tmpfile.write(text)` line?

Reply:
Alternatively I could update the comment to `Write script to temporary file and execute it`

Reply:
That works. Thanks.